crates/nu-command/tests/commands/redirection.rs

```diff
@@ -171,6 +171,22 @@ fn redirection_keep_exit_codes() {
     });
 }
 
+#[test]
+fn redirection_stderr_with_failed_program() {
+    Playground::setup("redirection stderr with failed program", |dirs, _| {
+        let out = nu!(
+            cwd: dirs.test(),
+            r#"$env.FOO = "bar"; nu --testbin echo_env_stderr_fail FOO e> file.txt; echo 3"#
```

Comment:
Was confused at first why `FOO` was passed bare but then I read the `echo_ENV_stderr_fail` 😆